torch/fx/graph.py

```diff
@@ -330,7 +330,14 @@ def register_modules_used(qualified_name : str):
 
         def type_repr(o : Any):
             typename = _type_repr(o)
-            register_modules_used(typename)
+            if all(x.isidentifier() for x in typename.split('.')):
```

Comment:
Not sure if I like this conditional but from my previous experience it seems like there's no good way to check if an object is a `typing` type object that works across Python versions